### PR TITLE
Initial cut of getAccountReportingHistory getter

### LINKED ISSUE #1808: Implement getAccountReportingHistory getter & tests
Reporting Information needed for market cards in reporting,  participation tokens and disputing form.

Input:
 - user account address
  - universe address

Outputs:
  summary: This is all user account specific information. All reporting/disputing stake information and participation information. This includes:
 - Profit and Loss percentage on reporting/disputing (REP staked / REP redeemed)
  Include REP profit in attoREP UI will format it.
 - All `IReportingParticipant` contracts and amounts the user has currently contributed to or can be redeemed. (don't include finalized markets user lost)
 - All dispute window contracts the user currently has participation tokens. This includes current disputing window and dispute windows in the past. 

* Result shape could be:
```typescript
{
  profitloss: decimal,
  profitAmount: string,
  reporting: {
    totalAmount: string;
    contracts: [{
      address: string,
      amount: string,
      marketId: string 
    }]
  },
  disputing: {
    totalAmount: string;
    contracts: [{
      address: string,
      amount: string,
      marketId: string 
    }]
  },
  participationTokens: {
    totalAmount: string;
    totalFees: string;
    contracts: [{
     address: string,
     amount: string,
   }]
 }
}
```
image

Screen Shot 2019-08-29 at 11 07 27 AM

### PR DESCRIPTION
**Pull request by justinbarry**
Doesn't close that tix
 #1808

### CODE REVIEW

**Review by bthaile — APPROVED** [review]
For what modifying the log. Looks good to me.

**Review by nuevoalex — APPROVED** [review]
👍

**Review by bthaile — APPROVED** [review]
Lgtm

File: packages/augur-sdk/src/api/Events.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ export class Events {
     this.provider.storeAbiData(abi.Augur as Abi, "Augur");
   }
 
-  async getLogs(eventName: string, fromBlock: number, toBlock: number, additionalTopics?: Array<string | string[]>): Promise<ParsedLog[]> {
+  async getLogs(eventName: string, fromBlock: number, toBlock: number | "latest", additionalTopics?: Array<string | string[]>): Promise<ParsedLog[]> {
```
Comment (bthaile) [inline packages/augur-sdk/src/api/Events.ts:15]: not a fan of mixing types number and string seems odd. Is there any other approach available?

**Pull request merged**